modules/nf-core/dragen/main.nf

```diff
@@ -0,0 +1,78 @@
+process DRAGEN {
+    tag "$meta.id"
+    label 'process_long'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE':
```

Comment:
not sure what to do about the container images - there is a docker one but it is not official and I haven't found the Dockerfile for it. One could properly create a docker and singularity image from that if we could get it somehow and put it into the nf-core/biocontainers registry - not sure which one it is.